assets/js/blocks/attribute-filter/block.js

```diff
@@ -26,7 +26,7 @@ import { getTaxonomyFromAttributeId } from '../../utils/attributes';
  * Component displaying an attribute filter.
  */
 const AttributeFilterBlock = ( { attributes } ) => {
-	const [ options, setOptions ] = useState( [] );
+	const [ options, setOptions ] = useState( null );
```

Comment:
hmm, why did you change this to null?  Have you verified any place assuming options is an array is checking it is an array first?

Reply:
Actually I abandoned a check I added this for so I will revert.